int32 length
int32 start_index
google_chat_ros/User user
## START AnnotationType ##
bool add
bool mention
bool slash_command
### END AnnotationType ###
